js/index.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
```

Comment:
Nice job refactoring to use JavaScript classes 👍. One problem is introduced while refactoring. I have checked your previous project in which you used objects for this project and it successfully `add book`, `remove book`, and store the values to local storage and retrieve back. After refactoring the application successfully work `add book`, `remove book`, and it stores the value in local storage when the use `add new` book but after removing the book and reloading the page the removed book is displayed back on the page this shouldn't have to happen. Kindly, fix this bug to update the local storage on `remove book`.